notebook2/notebook/notebook.py

```diff
@@ -357,11 +492,13 @@ def worker_image():
 @sockets.route('/wait')
 @requires_auth(for_page = False)
 def wait_websocket(ws):
-    pod_name = session['pod_name']
-    svc_name = session['svc_name']
-    jupyter_token = session['jupyter_token']
-    log.info(f'received wait websocket for {svc_name} {pod_name}')
-    # wait for instance ready
+    notebook = session['notebook']
+
+    if notebook is None:
+        return
```

Comment:
If the user opens a web socket without a notebook in the session, that's an error, no? We should let the web socket 500 and print a stack trace to the notebook logs.